lib/bolt/config/transport/base.rb

```diff
@@ -7,6 +7,30 @@ module Bolt
   class Config
     module Transport
       class Base
+        RUN_AS_OPTIONS = {
+          "run-as" => { type: String,
+                        external: true,
```

Comment:
I don't love that the `external` key (which is specific the SSH transport) is here in the base class, but I don't have a good alternative.

Reply:
We could do something like merge `external => true` in for these in the Rakefile, but  I don't think that's better.